src/tests/ftest/util/ior_test_base.py

```diff
@@ -59,6 +60,8 @@ def setUp(self):
         self.ior_cmd = IorCommand()
         self.ior_cmd.get_params(self)
         self.processes = self.params.get("np", '/run/ior/client_processes/*')
+        self.co_prop = self.params.get("container_properties",
```

Comment:
Ravi can't you do this in test_utils.TestContainer and make it a Basic Parameter?

Reply:
Probably, we can have to do individual parameters and assemble it as self.co_prop structure. Whatever be the case, it has to converted to a structure (DaosContProperties: daos/src/client/pydaos/raw/daos_api.py)